src/dispatch/plugins/dispatch_slack/incident/interactive.py

```diff
@@ -138,27 +141,39 @@ def configure(config):
     ]
 
     # don't need an incident context
-    app.command(config.slack_command_list_incidents, middleware=[db_middleware])(
-        handle_list_incidents_command
-    )
     app.command(config.slack_command_report_incident, middleware=[db_middleware])(
         handle_report_incident_command
     )
 
     # non-sensitive-commands
-    app.command(config.slack_command_list_tasks, middleware=middleware)(handle_list_tasks_command)
-    app.command(config.slack_command_list_my_tasks, middleware=middleware)(
-        handle_list_tasks_command
-    )
-    app.command(config.slack_command_list_participants, middleware=middleware)(
-        handle_list_participants_command
-    )
+    app.command(
+        config.slack_command_list_incidents,
+        middleware=middleware,
+    )(ack=ack_command, lazy=[handle_list_incidents_command])
```

Comment:
This prevents `list-incidents` from running outside of incident channels. 

You'll get:

```
Sorry, I can't determine the correct context to run the command /dispatch-list-incidents. Are you running this command in an incident channel?
```

This is caused by the `command_context_middleware`. Which we're using to get the org_slug to refetch the db. Need a better way here.